gentoo/gentoo
# gnome-base/librsvg: Version bump to 2.57.0

746 Test passed
1 failing

```
failures:

---- tests::bugs_bug668_small_caps_svg stdout ----
bug668-small-caps: 3551 pixels changed with maximum difference of 255
out: /var/tmp/portage/gnome-base/librsvg-2.57.0/work/librsvg-2.57.0-abi_x86_32.x86/tests/output/bug668-small-caps-out.png
diff: /var/tmp/portage/gnome-base/librsvg-2.57.0/work/librsvg-2.57.0-abi_x86_32.x86/tests/output/bug668-small-caps-diff.png
thread 'tests::bugs_bug668_small_caps_svg' panicked at 'surfaces are too different', rsvg/src/test_utils/reference_utils.rs:89:25
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    tests::bugs_bug668_small_caps_svg

test result: FAILED. 746 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 1.23s

error: test failed, to rerun pass `-p librsvg --test reference`
make[3]: *** [Makefile:1617: check-local] Error 101
make[3]: Leaving directory '/var/tmp/portage/gnome-base/librsvg-2.57.0/work/librsvg-2.57.0-abi_x86_32.x86'
make[2]: *** [Makefile:1455: check-am] Error 2
make[1]: *** [Makefile:1159: check-recursive] Error 1
make: *** [Makefile:1457: check] Error 2
 * ERROR: gnome-base/librsvg-2.57.0::gentoo failed (test phase):
 *   Make check failed. See above for details.
 *
 * Call stack:
 *            ebuild.sh, line  136:  Called src_test
 *          environment, line 3850:  Called multilib-minimal_src_test
 *          environment, line 2680:  Called multilib_foreach_abi 'multilib-minimal_abi_src_test'
 *          environment, line 2888:  Called multibuild_foreach_variant '_multilib_multibuild_wrapper' 'multilib-minimal_abi_src_test'
 *          environment, line 2570:  Called _multibuild_run '_multilib_multibuild_wrapper' 'multilib-minimal_abi_src_test'
 *          environment, line 2568:  Called _multilib_multibuild_wrapper 'multilib-minimal_abi_src_test'
 *          environment, line  579:  Called multilib-minimal_abi_src_test
 *          environment, line 2676:  Called default_src_test
 *   phase-functions.sh, line  901:  Called __eapi0_src_test
 *     phase-helpers.sh, line  832:  Called die
 * The specific snippet of code:
 *   		${emake_cmd} ${internal_opts} check || \
 *   			die "Make check failed. See above for details."

```
